security/permissions.rst

```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
```

Comment:
The access policy decisions are based on HL7 SMART on FHIR scopes, both SMART on FHIR v1 and v2 scopes are supported.